static/js/elementContentCleaner.js

```diff
@@ -14,17 +14,22 @@ elementContentCleaner.prototype.deleteElement = function() {
   var lineIsSceneMark = !utils.lineIsScriptElement(currentLine);
   if (lineIsSceneMark) return;
 
+  var lineToSelect;
   var lineIsHeading = utils.lineIsHeading(currentLine, this.attributeManager);
   if (lineIsHeading) {
     var $lineToDelete = this._getParentSMOfHeading(currentLine);
     var sceneMarkLineId = $lineToDelete.attr('id');
     removeSceneMark(sceneMarkLineId);
+    var titleAndSummaryLines = 2; // 2 lines
```

Comment:
cara, isso aqui me pareceu bem perigoso. E se tiver mais de um sumário? Se for uma sequência, ato, episódio e for deletar esse código é chamado? Não quebrou nada com isso?